src/item.h

```diff
@@ -1469,6 +1469,16 @@ class item : public visitable<item>
          * character that wears the item.
          */
         units::volume get_storage() const;
+        /**
+         * Returns the weight capacity modifier (@ref islot_armor::weight_capacity_modifier) that this item provides when worn.
+         * For non-armor it returns 1. The modifier is multiplied with the weight capacity of the character that wears the item.
+         */
+        float get_weight_capacity_modifier() const;
+        /**
+         * Returns the weight capacity bonus (@ref islot_armor::weight_capacity_modifier) that this item provides when worn.
+         * For non-armor it returns 1. The bonus is added to the total weight capacity of the character that wears the item.
```

Comment:
> For non-armor it returns 1.

Code says something else: it returns 0 there.

Reply:
copy paste mistake